sql/pgtap.sql.in

```diff
@@ -5924,11 +5924,16 @@ RETURNS TEXT[] AS $$
           JOIN pg_catalog.pg_namespace n ON p.pronamespace = n.oid
          WHERE n.nspname = $1
            AND p.proname ~ $2
+           AND ($3 IS NULL OR p.proname !~ $3)
```

Comment:
Does this work when `$3` is `NULL`? IME, I've had to use `CASE WHEN` to keep it from evaluating both sides of the `OR` statement when `$3` is `NULL`. Something like:

    AND CASE WHEN $3 IS NULL THEN TRUE ELSE p.proname !~ $3 END